src/CircleOcticon.js

```diff
@@ -6,10 +6,10 @@ import theme from './theme'
 import BorderBox from './BorderBox'
 
 function CircleOcticon(props) {
-  const {size, is} = props
+  const {size, as} = props
   const {icon, bg, ...rest} = props
   return (
-    <BorderBox is={is} bg={bg} overflow="hidden" border="none" size={size} borderRadius="50%">
+    <BorderBox as={as} bg={bg} overflow="hidden" border="none" size={size} borderRadius="50%">
       <Flex {...rest} alignItems="center" justifyContent="center">
```

Comment:
It looks like both `BorderBox` and `Flex` will get the same `as` prop. Is that intentional?

Reply:
Good call, it should only be passed down to BorderBox here